Doc/whatsnew/3.13.rst

```diff
@@ -748,6 +734,23 @@ base64
   See the `Z85 specification <https://rfc.zeromq.org/spec/32/>`_ for more information.
   (Contributed by Matan Perelman in :gh:`75299`.)
 
+
+concurrent.futures
+------------------
```

Comment:
concurrent.futures goes after compileall.

whitespace nit: the existing headings have a single new line separating them:

```suggestion
concurrent.futures
------------------
```

(and below)

Reply:
I'll be adding the extra blank lines in the Improved Modules PR for every section, though it does look odd currently. Will fix the order shortly.